src/xcode/ENA/ENA/Resources/Info_Debug.plist

```diff
@@ -56,6 +56,8 @@
 	<true/>
 	<key>UILaunchStoryboardName</key>
 	<string>LaunchScreen</string>
+	<key>UIMainStoryboardFile</key>
```

Comment:
Do we need this change in the other Info_*.plist as well?

Reply:
Good point!

Reply:
done